index.html

```diff
@@ -26,5 +26,19 @@ <h1>EDI Final Group Project</h1>
       crossorigin="anonymous"
     ></script>
     <script src="main.js"></script>
+   
```

Comment:
proszę obuduj to w sekcję i błagam zainstaluj proponowane dodatki w vscode, po tym by kod ładnie się formatował